moc_openstack/templates/surupd.erb

```diff
@@ -0,0 +1,3 @@
+wget -q http://<%= scope.function_hiera(['quickstack::params::repo_server']) %>/misc/emerging.rules.tar.gz > /dev/null && tar -xvzf emerging.rules.tar.gz -C /etc/suricata/ > /dev/null && rm -rf emerging.rules.tar.gz && sed -i s/alert/drop/g /etc/suricata/rules/drop.rules && sed -i s/alert/drop/g /etc/suricata/rules/emerging-scan.rules && sed -i /2006546/d /etc/suricata/rules/emerging-scan.rules && sed -i /2018754/d /etc/suricata/rules/emerging-scan.rules && sed -i /2210029/d /etc/suricata/rules/stream-events.rules && sed -i /2210045/d /etc/suricata/rules/stream-events.rules ||exit
```

Comment:
Is there any way this line can be shortened?  I believe each individual command can get a `|| exit`, and it'll work the same.